oppijanumerorekisteri-domain/src/main/java/fi/vm/sade/oppijanumerorekisteri/repositories/HenkiloRepository.java

```diff
@@ -20,6 +20,6 @@
 
     Optional<Henkilo> findByOidHenkilo(String henkiloOid);
 
-    Optional<Henkilo> findByHetu(String hetu);
+    Optional<Henkilo> findFirstByHetu(String hetu);
```

Comment:
Eikös tämä ole aika vaarallinen? Jos tietokannasta syystä tai toisesta löytyy samalla henkilötunnuksella rivi (näin ei toki pitäisi koskaan olla), palautetaan niistä jompi kumpi, riippuen tietokannan olotilasta. Mielestäni oikea tapa on heittää poikkeus (kuten vanha toteutus) ja virkailijan tulee korjata tilanne.